docs/openapi/oad/amphora.oad.yaml

```diff
@@ -0,0 +1,856 @@
+openapi: 3.1.0
+info:
+  title: Amphora Service API
+  description: |
+    This API allows to interact with the Amphora service. It provides functionality to store, fetch and manage secret
+    shared data as used for MPC-based distributed computation on encrypted data in the Carbyne Stack platform.
+
+    The endpoints provided by Amphora are divided into three categories:
+    - User facing: \
+      Used by clients for secret sharing, accessing, and managing secret shares and their tags (*/input-masks*,
```

Comment:
What is the point of enumerating the paths here?

Reply:
removed